server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -276,15 +276,19 @@ def get_sequence(self, filepath, pattern="%04d"):
         """
         import clique
 
-        escaped = re.escape(filepath)
+        escaped = (
+            re.escape(os.path.basename(filepath))
+            if os.path.isabs(filepath) else re.escape(filepath)
+        )
         re_pattern = escaped.replace(pattern, "-?[0-9]+")
 
         source_dir = os.path.dirname(filepath)
         files = [f for f in os.listdir(source_dir)
                  if re.match(re_pattern, f)]
 
         pattern = [clique.PATTERNS["frames"]]
-        collection, remainder = clique.assemble(files, patterns=pattern)
+        collection, remainder = clique.assemble(
+            files, patterns=pattern, minimum_items=1)
```

Comment:
This still feels off because also for "relative paths" the match would fail if it contains folders.
E.g. this would on a path like `hello/world/image.<UDIM>.exr` or `../other/folder/texture.<UDIM>.exr`. 

We should always just compare to the filename, e.g. something like:
```suggestion
        filename = os.path.basename(filepath)
        re_pattern = re.escape(filename)
        re_pattern = re_pattern.replace(re.escape(pattern), "-?[0-9]+")

        source_dir = os.path.dirname(filepath)
        files = [f for f in os.listdir(source_dir) if re.match(re_pattern, f)]

        collection, _remainder = clique.assemble(
            files, 
            patterns=[clique.PATTERNS["frames"]], 
            minimum_items=1)
```